components/form/LinkInverters.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import InverterGraphicIcon from '../icons/InverterGraphicIcon';
+import Link from 'next/link';
+import useMediaQuery from '~/lib/hooks/useMediaQuery';
+import Button from '../Button';
+import Spinner from '../Spinner';
+import { ChevronRightIcon } from '@heroicons/react/24/solid';
+
+const LinkInverters = () => {
+  const [showDisclaimer, setShowDisclaimer] = useState(false);
+  const mobile = useMediaQuery('(max-width: 768px)');
+
+  const mobileSkipButtonClass =
+    'w-full text-ocf-yellow nderline text-[14px] mt-[5px]';
+
+  const mobileInverterLinkClass =
+    'h-[54px] w-[308px] bg-ocf-yellow rounded-md font-semibold mb-[20px]';
+
+  const deslktopInverterLinkClass =
```

Comment:
Change references below
```suggestion
  const desktopInverterLinkClass =
```